server/src/main/java/org/elasticsearch/index/mapper/vectors/SparseVectorFieldMapper.java

```diff
@@ -92,11 +92,6 @@ public String typeName() {
             return CONTENT_TYPE;
         }
 
-        @Override
-        public Query existsQuery(SearchExecutionContext context) {
-            throw new IllegalArgumentException("[sparse_vector] fields do not support [exists] queries");
-        }
```

Comment:
If the field was created in an 8.11 index, we should probably still throw here. We never want to get in the scenario where some `sparse_vector` match `exists` and others do not in the same index. 

That would be confusing.

Reply:
NOTE: This is irrelevant of 8.11 or not. Its a NEW commit and a new change. We should handle this correctly given the new `IndexVersion` format.

Reply:
Yeap, makes sense. I was considering that "some" results would still be valid, but the response will be inconsistent. Will update as suggested.